src/main.cpp

```diff
@@ -1,14 +1,16 @@
-#include <vector>
-#include <iostream>
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+ #include <iostream>
+ #include <vector>
+ #include "functions.hpp"
```

Comment:
it's better when include header is on the top

Reply:
It's good according to Mateusz:

https://coders.school/edu/courses/powtorka-1-qa/lesson/kolejnosc-include/